examples/custom-action-offload-to-s3/firstLambda.js

```diff
@@ -0,0 +1,64 @@
+const AWS = require('aws-sdk');
+const fetch = require("node-fetch");
+
+async function fetchDataFromAPI (id) {
```

Comment:
I would revise this function name to be `fetchAsset` because `fetchDataFromAPI` is very broad.

Reply:
Good call